inference-engine/thirdparty/clDNN/kernel_selector/core/cl_kernels/convolution_gpu_bfyx_to_bfyx_f16.cl

```diff
@@ -89,7 +97,7 @@ KERNEL(convolution_bfyx_to_bfyx_f16)(
     bias_offset += split_idx * BIAS_LENGTH;
 #   endif
 
-    MAKE_VECTOR_TYPE(INPUT0_TYPE, OUTPUT_X_BLOCK_SIZE) dst = (MAKE_VECTOR_TYPE(INPUT0_TYPE, OUTPUT_X_BLOCK_SIZE))(DT_INPUT_BLOCK_READ(biases, bias_offset));
+    MAKE_VECTOR_TYPE(INPUT0_TYPE, OUTPUT_X_BLOCK_SIZE) dst = (MAKE_VECTOR_TYPE(INPUT0_TYPE, OUTPUT_X_BLOCK_SIZE))(DT_BIAS_BLOCK_READ(biases, bias_offset));
```

Comment:
Could you describe more in detail of commit message? For example, what is changed and why it is changed.

Reply:
It's a bug due to bias data type mismatch. I am going to add details next update.

Reply:
It is a bug due to bias data type mismatching and might still affect to openvino/master. I will describe more in commit message.